client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/theme-info-popup.scss

```diff
@@ -1,123 +0,0 @@
-@import "@wordpress/base-styles/breakpoints";
```

Comment:
Removed as this is no longer in use since we removed the `ThemeInfoPopup` component in this PR.